templates/kube-scheduler.yaml

```diff
@@ -13,10 +13,23 @@ spec:
     - scheduler
     - --master=http://127.0.0.1:8080
     - --leader-elect=true
+    {% if kube_scheduler_policy_config_dir is defined -%}
+    - --policy-config-file={{kube_scheduler_policy_config_dir}}/{{kube_scheduler_policy_config_file}}
```

Comment:
Create another variable called kube_scheduler_policy_config_file_abs = "{{kube_scheduler_policy_config_dir}}/{{kube_scheduler_policy_config_file}}" in defaults/main.yaml since you are using it in a couple of places.